### ISSUE TITLE
有没有可能通过docker配置plugin

### ISSUE THREAD

**Issue by iswangyk** [ChatGPTNextWeb/NextChat-Awesome-Plugins#8]
有没有可能通过docker配置plugin，这样切换设备就不用重新配置了

**Comment by lloydzhou** [comment]
这里的插件数据存储在 store中， 只要保证这个store的数据能同步，就可以跨设备使用

**Comment by iswangyk** [comment]
> 这里的插件数据存储在 store中， 只要保证这个store的数据能同步，就可以跨设备使用

Thanks @lloydzhou 有点疑惑，怎么找到这个store呢，我是用docker部署，本地用浏览器打开，是在cookie里吗

**Comment by lloydzhou** [comment]
这个store之前在locastorage里面，现在应该在indexeddb里面

**Comment by lloydzhou** [comment]
现在有一个内置插件的方式：
1. 在v2.15.3版本支持通过`/plugins.json`拿到一个内置的插件列表，进行初始化。
2. 所以，你部署的时候，只要将你自己的plugins.json放到 static目录里面覆盖掉应该就可以了。

**Comment by LoopOptimum** [comment]
> Contributor

我用docker-compose 部署，把自己的 plugins.json 映射 也成功了
    volumes:
      - ./public:/app/public
不过只能内置不需要令牌的插件，诸如 wolframalpha，怎样内置自己的令牌跨设备使用呢

**Comment by lloydzhou** [comment]
我看你在 #20 里面已经解决了

**Comment by LoopOptimum** [comment]
> 我看你在 #20 里面已经解决了

对。另一个问题是对于没有托管在 github的schema (openai.json / openai.yaml), 也是无法通过plugin.json 内置 ; 比如gapier 和 webpilot.  
我只能Fork你的把这俩加上去

**Comment by silenceroom** [comment]
> > 现在有一个内置插件的方式：
> > 
> > 1. 在v2.15.3版本支持通过`/plugins.json`拿到一个内置的插件列表，进行初始化。
> > 2. 所以，你部署的时候，只要将你自己的plugins.json放到 static目录里面覆盖掉应该就可以了。
> 
> [@lloydzhou](https://github.com/lloydzhou) 只能在 2.15.3版本么？

没有问题了，是原始项目中的 plugins.json引用的链接的问题。
